packages/trial/middleware.js

```
@@ -2,8 +2,9 @@ import {
   actions as dataFetchActions,
   actionTypes as dataFetchActionTypes,
 } from '@bufferapp/async-data-fetch';
+import { actionTypes as orgActionTypes } from '@bufferapp/publish-data-organizations';
 import { actions as notificationActions } from '@bufferapp/notifications/lib/';
-import { AppHooks } from '@bufferapp/publish-composer/composer/utils/LifecycleHooks';
+import AppHooks from '@bufferapp/publish-composer/composer/utils/LifecycleHooks';
```

This one is interesting, as this import was not working at all

---

😮 Very interesting... was it just failing silently?

---

Yep, when users started a pro trial through an upgrade path (for instance the first comment one), the composer was not updating the plan info because AppHooks was not defined 😅

---

Oooh, I feel like I noticed this a loooong time ago. 😅   Good catch! 👍 💯

---

Found it! https://buffer.atlassian.net/browse/PUB-2205

You should claim that and mark as Fixed! Sweet sweet Jira karma 😂